crates/router/src/core/errors/api_error_response.rs

```diff
@@ -319,7 +319,7 @@ impl common_utils::errors::ErrorSwitch<api_models::errors::types::ApiErrorRespon
             Self::ClientSecretNotGiven => AER::BadRequest(ApiError::new(
                 "IR",
                 8,
-                "Client secret was not provided", None
+                "client_secret was not provided", None
```

Comment:
If we're mentioning field names, can we enclose them in backticks? We can make this change across all error messages in a separate PR if necessary.

So the error message would be:
```
`client_secret` was not provided
```